versions/cn/toml-v0.5.0.md

```diff
@@ -0,0 +1,802 @@
+![TOML Logo](../../logos/toml-200.png)
+
+TOML v0.5.0
+===========
+
+汤小明的小巧明晰语言。
```

Comment:
不要调侃，这是一份严肃的文档。
建议和0.4的一致吧。 

全称: Tom的（语义）明显、（配置）最小化的语言。 (Tom's Obvious, Minimal Language. By Tom Preston-Werner.)

Reply:
此处是所有指正中唯一希望得到进一步探讨的地方。TOML 的命名中使用了双关语，Obvious 和 Minimal 两个词虽然是 TOML 的重要特点，但具体选择这两个单词肯定是和 Tom 一词的后两个字母脱不开关系的。因此，TOML 一词既可以理解为 Tom's Obvious, Minimal Language，又可以理解为 Tom Language，可谓独具匠心，如不能在译文中传达，实属遗憾。
@ZhangHanDong 谢谢！希望此处能讨论得出一个既正式又不失灵巧的译法

Reply:
@pradyunsg Here is the only place of all correction where further discussion is desired.

Pun is used in the naming of TOML. Although the two words Obvious and Minimal are important features of TOML, the specific choice of these two words is bound to be related to the last two letters of Tom.

Therefore, the word TOML can be understood as either Tom's Obvious, Minimal Language or Tom Language, which is very ingenious. It is a pity that it cannot be conveyed in the translation.

My original translation retains the punny meaning, but a little informal. The edition by proofreader is more official, but drop the pun.

I think before we get an integrated solution in the future, we need to decide to use which now. Maybe it depends on the style of the spec maintainers team, because there are no right or wrong matter.

Reply:
个人觉得 直接用最直白的翻译就行了，技术文档主要还在于讲清楚技术层面的事情。Tom的（语义）明显、（配置）最小化的语言

本身TOML 来源于英文的缩略，精巧一些也是难免。:smily:

Reply:
Let's translate as "Tom's Obvious Minimal Language", dropping the pun in preference to being more formal.